conf/application.conf

```diff
@@ -183,7 +183,7 @@ baseUrl {
   accessibility-statement-frontend-local = "http://localhost:12346"
   direct-debit-update-email-frontend-local = "http://localhost:10801"
   gg = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  sign-out = "http://localhost:9949/auth-login-stub/session/logout"
+  sign-out = "http://localhost:9553"
```

Comment:
can we call this something like `sign-out-local` so we know we don't need to update it in the app-configs?

Reply:
ah yeah good shout!

Reply:
Done